sw/device/lib/testing/test_framework/ottf.c

```diff
@@ -85,6 +87,14 @@ int main(int argc, char **argv) {
     init_uart();
   }
 
+  if (kJitterEnabled) {
```

Comment:
minor nit: could we add a small comment here something like:
```c
// The kJitterEnabled symbol defaults to false across all hardware platforms.  However, in DV
// simulation, it may be overridden via a backdoor write with the plusarg: `+en_jitter=1`.
```

Reply:
Added, thanks